app/src/main/java/com/keylesspalace/tusky/components/preference/AccountPreferencesFragment.kt

```diff
@@ -198,7 +200,6 @@ class AccountPreferencesFragment : PreferenceFragmentCompat(), Injectable {
                     setOnPreferenceChangeListener { _, newValue ->
                         setIcon(getIconForVisibility(Status.Visibility.byString(newValue as String)))
                         syncWithServer(visibility = newValue)
-                        eventHub.dispatch(PreferenceChangedEvent(key))
```

Comment:
`PreferencesFragment` send the event everytime a pref changes, so these were redundant

Reply:
I am a bit confused by your comment, where does it do that?